src/views/Generator/RuleEditor/CorrelationEditor.tsx

```diff
@@ -5,9 +5,21 @@ import { FilterField } from './FilterField'
 import { SelectorField } from './SelectorField'
 import { Label } from '@/components/Label'
 import { useFormContext } from 'react-hook-form'
+import { FieldGroup } from '@/components/Form'
+import { ControlledRadioGroup } from '@/components/Form/ControllerRadioGroup'
+
+const EXTRACTION_MODE_OPTIONS = [
+  { value: 'single', label: 'First occurence' },
+  { value: 'multiple', label: 'All occurences' },
+]
```

Comment:
I've been thinking about what @allansson said the other day and I think this could indeed be improved.
Maybe a switch with two labels: `Use first value` and `Use latest value`? It'd also be a better description of how the rule actually works (`all` is too ambiguous IMO).

What do you think?

Reply:
Not sure if it's easy to understand with `first` and `latest` value 🤔 
Same for `all`, although it seems to make it clearer on a conceptual level (more than one) I agree it might not be the best.
I would say, let's gather more feedback 🤔